Dockerfile

```diff
@@ -10,26 +10,27 @@ WORKDIR /stk
 ENV VERSION=1.1
 
 # Install build dependencies
-ENV DEBIAN_FRONTEND=noninteractive
+ARG DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && \
-    apt-get install -y build-essential \
+    apt-get install --no-install-recommends -y build-essential \
                        cmake \
                        git \
                        libcurl4-openssl-dev \
                        libenet-dev \
                        libssl-dev \
                        pkg-config \
                        subversion \
-                       zlib1g-dev
+                       zlib1g-dev \
+                       ca-certificates
 
 # Get code and assets
 RUN git clone --branch ${VERSION} --depth=1 https://github.com/supertuxkart/stk-code.git
-RUN svn checkout https://svn.code.sf.net/p/supertuxkart/code/stk-assets-release/${VERSION}/ stk-assets
```

Comment:
Versions are important for getting reproducible builds. Since STK 1.2 does not include updated assets, `VERSION` could be split up in `CODE_VERSION` and `ASSET_VERSION`.